src/VarDumper.php

```diff
@@ -134,27 +245,29 @@ private static function dumpInternal($var, int $level): void
         }
     }
 
-    /**
-     * Exports a variable as a string representation.
-     *
-     * The string is a valid PHP expression that can be evaluated by PHP parser
-     * and the evaluation result will give back the variable value.
-     *
-     * This method is similar to `var_export()`. The main difference is that
-     * it generates more compact string representation using short array syntax.
-     *
-     * It also handles objects by using the PHP functions serialize() and unserialize().
-     *
-     * PHP 5.4 or above is required to parse the exported value.
-     *
-     * @param mixed $var the variable to be exported.
-     * @return string a string representation of the variable
-     */
-    public static function export($var): string
+    private static function getVarDumpValuesArray($var): array
     {
-        self::$output = '';
-        self::exportInternal($var, 0);
-        return self::$output;
+        if ('__PHP_Incomplete_Class' !== get_class($var) && method_exists($var, '__debugInfo')) {
```

Comment:
I think will be nice to flip the condition. WDYT?

Reply:
For what purpose?